webapp/graphite/storage.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, finders=None, hosts=None):
 
     if hosts is None:
       hosts = settings.CLUSTER_SERVERS
-    remote_hosts = [host for host in hosts if not is_local_interface(host)]
+    remote_hosts = [host for host in hosts if not settings.REMOTE_EXCLUDE_LOCAL or not is_local_interface(host)]
```

Comment:
Doing some testing locally, it looks like defaulting REMOTE_EXCLUDE_LOCAL to 'False' is the inverse of what we have today, no?
```python
hosts = ['onehost', 'twohost', 'threehost', 'localhost', 'greenhost']

def check_item(item):
  print("Checking if localhost: ", item)
  if item == 'localhost':
    return True
  else:
    return False

print([host for host in hosts if not check_item(host)])

EXCLUDE_LOCAL = True

print([host for host in hosts if not EXCLUDE_LOCAL or not check_item(host)])

EXCLUDE_LOCAL = False

print([host for host in hosts if not EXCLUDE_LOCAL or not check_item(host)])
```

ouputs:

```
('Checking if localhost: ', 'onehost')
('Checking if localhost: ', 'twohost')
('Checking if localhost: ', 'threehost')
('Checking if localhost: ', 'localhost')
('Checking if localhost: ', 'greenhost')
['onehost', 'twohost', 'threehost', 'greenhost']
('Checking if localhost: ', 'onehost')
('Checking if localhost: ', 'twohost')
('Checking if localhost: ', 'threehost')
('Checking if localhost: ', 'localhost')
('Checking if localhost: ', 'greenhost')
['onehost', 'twohost', 'threehost', 'greenhost']
['onehost', 'twohost', 'threehost', 'localhost', 'greenhost']
```

Would we not want this to be backwards compatible by default?